doc/api/vm.markdown

```diff
@@ -301,6 +301,38 @@ e.g. `(0,eval)('code')`. However, it also has the following additional options:
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
 
+## Example: Run a Server within a VM
+
+The context of `.runInThisContext()` refers to the v8 context. The code passed
+to this VM context will have it's own isolated scope. To run a simple web server
+using the `http` module, for instance, the code passed to the context must either
+call `require('http')` on it's own, or have a reference to the `http` module passed
+to it. For instance:
+
+```js
+'use strict'
```

Comment:
In our repo every piece of code will have semicolons to mark the end of line. This example doesn't conform to that practice